Misc/NEWS.d/next/Library/2024-10-22-19-52-15.gh-issue-87063.fR-wJa.rst

```diff
@@ -0,0 +1,2 @@
+Add the :meth:`.is_closed()` in the :class:`multiprocessing.Process` to
```

Comment:
Please, do not use the `.`-trick for links in NEWS entries in general unless you know that there will not be any conflicts, and prefer using fully-qualified names + custom titles syntax or plain inline rendering with double backquotes if you don't want a link. You can also decide to render the fully-qualified names.

Examples:

- <code>:meth:\`is_closed <multiprocessing.Process.is_closed>\`</code>
- <code>:meth:\`multiprocessing.Process.is_closed\`</code> or
- <code>\`\`is_closed\`\`</code>

Note that in all of these cases, the new interface must be documented in the corresponding `.rst` file since some users may not look at the Python code itself.